src/form.js

```diff
@@ -225,7 +225,7 @@ export default React.createClass({
             (this.props.validators || []),
             (this.validators || [])
         );
-        const formErrors = formValidators.map(fn => fn(form));
+        const formErrors = formValidators.map(fn => fn(form)).filter(e => e != undefined);
```

Comment:
Form validators are broken as it stands. Validators returning nothing populate `errors` with undefined. This weeds them out.